model/tensorflow_hub/LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2019 himanshu
```

Comment:
I'm pretty sure we need this to be `Intel, Your full name`, since it's going in the intel repo.